Index: back_end_grado/http/grade_profile_has_task.http
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>### Get all the actives entries of grade profiles\nGET http://127.0.0.1:8080/api/v1/grade-profile-tasks/\n\n### Get all is current task by grade profiles\nGET http://127.0.0.1:8080/api/v1/grade-profile-tasks/tasks/\n\n### Get all task of a user\nGET http://127.0.0.1:8080/api/v1/grade-profile-tasks/user-tasks?idUsers=2
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/back_end_grado/http/grade_profile_has_task.http b/back_end_grado/http/grade_profile_has_task.http
--- a/back_end_grado/http/grade_profile_has_task.http	
+++ b/back_end_grado/http/grade_profile_has_task.http	
@@ -5,4 +5,4 @@
 GET http://127.0.0.1:8080/api/v1/grade-profile-tasks/tasks/
 
 ### Get all task of a user
-GET http://127.0.0.1:8080/api/v1/grade-profile-tasks/user-tasks?idUsers=2
\ No newline at end of file
+GET http://127.0.0.1:8080/api/v1/grade-profile-tasks/user-tasks?idUsers=972
\ No newline at end of file
